website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -63,23 +63,24 @@ Properties without a default value are required.
 | `data.attributes.source-url` **(beta)**       | string  | (nothing) | A URL for the application or client creating this workspace. This can be the URL of a related resource in another app, or a link to documentation or other info about the client.                                                                                                                                                                                                                                                                                                                               |
 | `data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the CLI or the runs API.                                                                                                                                                                                    |
 | `data.attributes.terraform-version`           | string  | (special) | The version of Terraform to use for this workspace. This can be either an exact version or a [version constraint](/language/expressions/version-constraints) (like `~> 1.0.0`); if you specify a constraint, the workspace will always use the newest release that meets that constraint. If omitted when creating a workspace, this defaults to the latest released version.                                                                                                                                   |
-| `data.attributes.trigger-prefixes`            | array   | `[]`      | List of repository-root-relative paths which should be tracked for changes, in addition to the working directory.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.trigger-prefixes`            | array   | `[]`      | List of trigger prefixes that describe the paths Terraform Cloud monitors for changes, in addition to the working directory. Trigger prefixes are always appended to the root directory of the repository.                                                                                                                                                                                                                                                                                                      |
```

Comment:
I think we should update this here to say something like "the prefix matching implies that any directory who's path matches the prefix set will cause a run to be triggered."

Reply:
ahhh. okay I think "Terraform Cloud will start a run when files are changed in any directory path matching the provided set of prefixes." 

@omarismail thoughts?

Reply:
Changed according to @laurapacilio's proposal. 771411637c386d91a2f7225babbffac3d9ea2ed5

cc. @omarismail